src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -155,10 +158,6 @@ export class EditDataGridPanel extends GridParentComponent {
 		// TODO add any Edit Data-specific shortcuts here
 	}
 
-	public onDestroy(): void {
```

Comment:
why removing this?

Reply:
It was never called at all in the code at all when I checked it, I decided to add it to the parent class's dispose function.